File: src/DataStructure/Array/1D/DominantIndex/python/DominantIndex_test.py
import pytest
from DominantIndex.python.dominantIndex import Solution
from DominantIndex.python.testCases import runTestCase, cases


@pytest.mark.parametrize('case', cases)
def test(case):
    solution = Solution()
    assert runTestCase(solution.dominantIndex, case) == case['expected']

File: src/DataStructure/Array/1D/DominantIndex/javascript/dominantIndex.test.js
import solution from './dominantIndex.es3.cjs';
import runTestCase, { cases } from './testCases.js';

test.each(cases.map(c => [c.name, c]))(
  '[Data Structure/Array/1D/[747] Largest Number At Least Twice of Others] - Case %#',
  (name, testCase) => expect(runTestCase(solution, testCase)).toEqual(testCase.expected),
);
